pkg/webhook/core.oam.dev/v1beta1/componentdefinition/validating_handler.go

```diff
@@ -97,6 +97,12 @@ func (h *ValidatingHandler) Handle(ctx context.Context, req admission.Request) a
 				return admission.Denied(err.Error())
 			}
 		}
+
+		version := obj.Spec.Version
+		err = webhookutils.ValidateVersionAndRevisionNameAnnotation(version, revisionName, obj.Kind)
```

Comment:
Can we rename it to `IsVersionAndRevisionNamePresnt`  and change function signature to return `bool` and `error`?